Loader/Loader/Program.cs

```diff
@@ -1,97 +1,103 @@
                     Console.WriteLine(e.Message);
+                    inputString = "";
                 }
 
-                //entering our data into a file
-                Console.WriteLine("Program exit:  ");
-                string writePath = @"C:\Users\Даша\source\repos\tmp2.txt";
+                //read data from SampleOutput
                 try
                 {
-                    using (StreamWriter sw = new StreamWriter(writePath, false, System.Text.Encoding.Default))
-                    {
-                        //sw.WriteLine("Дозапись");
-                        string C = Convert.ToString(c);
-                        string D = Convert.ToString(d);
-                        sw.Write(C);
-                        sw.Write(D);
-                    }
-                    using (StreamReader sr = new StreamReader(writePath))
-                    {
-                        Console.WriteLine(sr.ReadToEnd());
-                    }
-                    using (StreamReader sr = new StreamReader(writePath))
+                    using (StreamReader sr = new StreamReader(SampleOutput))
                     {
-                        text2=sr.ReadToEnd();
+                        outputString = sr.ReadToEnd();
+                        Console.WriteLine("Output String: ");
+                        Console.WriteLine(outputString);
+                        sr.BaseStream.Position = 0;
+                        ms.Position = 0;
+                        using (StreamReader srr = new StreamReader(ms))
+                        {
+                            ourOutputString = srr.ReadToEnd();
+                            Console.WriteLine("Our Output String: ");
+                            Console.WriteLine(ourOutputString);
+                        }
                     }
                 }
                 catch (Exception e)
                 {
                     Console.WriteLine(e.Message);
+                    ourOutputString = "";
+                    outputString = "";
                 }
-
-                //compare of two lines
-                if (text1 == text2) {
-                    return true;
-                }
-                else
-                {
-                    return false;
-                }
-                /*
-                //cравнение двух файлов сначала по длине, потом по байтам
-                if (new FileInfo(path).Length != new FileInfo(writePath).Length)
-                {
-                    return false;
-                }
-                else
-                {
-                    return (File.ReadAllBytes(path).SequenceEqual(File.ReadAllBytes(writePath)));
-                }
-                */
+                return (outputString == ourOutputString);
             }
             else 
-            { 
+            {
                 return false;
             }
-            
         }
 
-
-        static void Main(string[] args)
+        public static int[] gradingStudents(int[] grades)
```

Comment:
Move challenge in separate static class. Put this class in separate directory. And add txt files with input and output to this directory.